### PR TITLE
changes to Basic.free_symbols docstring

### PR DESCRIPTION
**Pull request by leosartaj**

### CODE REVIEW

File: sympy/core/basic.py [hunk 1]
```diff
@@ -484,13 +484,13 @@ def free_symbols(self):
 
         For most expressions, all symbols are free symbols. For some classes
         this is not true. e.g. Integrals use Symbols for the dummy variables
-        which are bound variables, so Integral has a method to return all symbols
-        except those. Derivative keeps track of symbols with respect to which it
+        which are bound variables, so Integral has a method to return all
+        symbols except those. Derivative keeps track of symbols with respect to which it
```
Comment (toolforger) [inline sympy/core/basic.py]: You wrapped the previous line but now this line is quite long.

**Comment by toolforger** [comment]
The title should characterize the change, so that people skimming a git log can decide whether a particular commit is interesting to them. The current title is too unspecific for that.

**Comment by smichr** [comment]
+1

**Comment by debugger22** [comment]
Failed build is a timeout. Merging it.

**Pull request merged**